doc/make.jl

```diff
@@ -371,21 +372,46 @@ else
 end
 
 const output_path = joinpath(buildroot, "doc", "_build", (render_pdf ? "pdf" : "html"), "en")
-makedocs(
+doc = makedocs(
     build     = output_path,
     modules   = [Main, Base, Core, [Base.root_module(Base, stdlib.stdlib) for stdlib in STDLIB_DOCS]...],
     clean     = true,
     doctest   = ("doctest=fix" in ARGS) ? (:fix) : ("doctest=only" in ARGS) ? (:only) : ("doctest=true" in ARGS) ? true : false,
     linkcheck = "linkcheck=true" in ARGS,
     linkcheck_ignore = ["https://bugs.kde.org/show_bug.cgi?id=136779"], # fails to load from nanosoldier?
-    checkdocs = :none,
+    checkdocs = :exports, # we really want :public but thats not available yet https://github.com/JuliaDocs/Documenter.jl/issues/1506
+    warnonly  = :missing_docs, # warn about missing docstrings, but don't fail
     format    = format,
     sitename  = "The Julia Language",
     authors   = "The Julia Project",
     pages     = PAGES,
     remotes   = documenter_stdlib_remotes,
+    debug     = true, # makes makedocs return the Documenter object for use later
 )
 
+
+const known_missing_from_manual = 299
+
+# Check that we're not regressing in missing docs, but only check on PRs so that master builds can still pass
+# Only check on Linux 64-bit because the number is platform-dependent
+if !in("deploy", ARGS) && Sys.islinux() && Sys.WORD_SIZE == 64
```

Comment:
Maybe only check the online manual build?

```suggestion
if in("deploy", ARGS) && Sys.islinux() && Sys.WORD_SIZE == 64
```

Reply:
Then PRs would be green and master commits red, and the docs not deployed. We want the opposite

Reply:
`BUILDKITE_BRANCH != "master"` is the way to tell if on a PR